portfolio/src/main/webapp/style.css

```diff
@@ -137,4 +137,80 @@ nav ul li a{
 	font-family: "Montserrat", sans-serif;
 	justify-content: center;
 	margin: 0 auto;
-}
\ No newline at end of file
+}
+
+.board{
+	width: 100%;
+	height: 100vh;
+	background-image: url("images/board.png");
+	background-repeat: no-repeat;
+	background-size: contain;
+	position: relative;
+	background-position: center;
+	padding-top: 60px;
+}
+
+.msg-send-button{
+	padding: 9px 1px 9px;
+	background-color: rgba(238,171,162,0.8);
+	border: none;
+	border-radius: 50px;
+	cursor: pointer;
+	transition: all 0.2s ease 0s;
+	font-family: "Montserrat", sans-serif;
+	font-weight: bold;
+	font-size: 18px;
+	width: 100px;
+	margin-top: 10px;
+	outline: none;
+ }
+
+.msg-send-button:hover {
+	background-color: rgba(238,171,162,1);
+}
+
+.text-field{
+	border: 1px solid #EEABA2;
+	border-radius: 50px;
+	height: 38px;
+	padding-left: 19px;
+	font-family: "Montserrat", sans-serif;
+	outline: none;
+	margin-top: 10px;
+	margin-right: 20px;
+	width: 87%;
+}
+
+.msg-text{
+	font-family: "Montserrat", sans-serif;
+	font-weight: bold;
+	color: #1a1a1a;
+	font-size: 20px;
+}
+
+.contact-form{
+	width: 90%;
+	margin: 90px auto 20px auto ;
+	position: relative;
+	align-items: left;
+}
+
+.postit{
+	position: absolute;
+	height: auto;
+	width: 10%;
+	display: none;
+	/*top: 25%;*/
+	/*left: 30%;*/
+	/*top: 62%;
+	left: 62%;*/
```

Comment:
Should this be removed? Looks like this is commented.

Reply:
Yes, this is commented out but I left it there on purpose. It is some information regarding the proper placement of the post-its on the board. Hence I thought of leaving it there for future reference.

Reply:
then can we add a comment on top stating these are left here on purpose? That way this will not be confused with dead/unused code...

Reply:
okay noted!